core/src/main/java/hudson/model/TopLevelItemDescriptor.java

```diff
@@ -114,6 +127,64 @@ public String getDisplayName() {
     }
 
     /**
+     * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
+     * in order to avoid how it should be represented.
+     *
+     * @return A string, an empty string by default.
```

Comment:
:ant: more accurate to say that the default is taken from the `newInstanceDetail` view.